Sources/Commands/SwiftPackageTool.swift

```diff
@@ -31,7 +31,22 @@ public struct SwiftPackageTool: ParsableCommand {
         abstract: "Perform operations on Swift packages",
         discussion: "SEE ALSO: swift build, swift run, swift test",
         version: SwiftVersion.currentVersion.completeDisplayString,
-        subcommands: [
+        subcommands: getSubCommands(),
+        helpNames: [.short, .long, .customLong("help", withSingleDash: true)])
+
+    @OptionGroup()
+    var swiftOptions: SwiftToolOptions
+
+    public init() {}
+
+    public static var _errorLabel: String { "error" }
```

Comment:
Is this a requirement of `swift-argument-parser`?

Reply:
We do need the constructor, but `_errorLabel` isn't needed. 
From the `swift-argument-parser` documentation `_errorLabel` is used to insert the label at the beginning of an error message.
```
/// The label to use for "Error: ..." messages from this type. (experimental)
    public static var _errorLabel: String { get }
```